src/containers/VideoItem.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
```

Comment:
This file is present in the other PR you made, is this supposed to be the case, or did you accidentally include a commit you shouldn't have?